doc/nrf/shortcuts.txt

```diff
@@ -60,6 +60,8 @@
    See :ref:`ug_zigbee` for more information.
 .. |zigbee_library| replace:: This is a Zigbee library.
    See :ref:`ug_zigbee_configuring_libraries` for information about how to configure it for you Zigbee application.
+.. |zigbee_ncp_package| replace:: It contains the full development-ready source code for host and evaluation-ready firmware for the nRF52840 dongle (nRF SoC; not subject to modification).
```

Comment:
There is firmware for nrf52840 DK too.

Reply:
Hmm, I was told by @tomchy that only dongle is supported.

Reply:
@tomchy we need your input

Reply:
It contains three firmware types:
 - Distribution package for the nRF52840 dongle
 - 2 firmware images for nRF52840 DK

That said, both nRF52840 dongle as well as nRF52840 DK are supported.